test/integration/breadcrumbs_switcher_test.rb

```diff
@@ -12,4 +12,15 @@ class BreadcrumbsSwitcherTest < IntegrationTestWithJavascript
       assert_equal current_bookmark_name, 'three'
     end
   end
+
+  test "search query" do
+    visit bookmarks_path
+    click_link 'foo'
+    click_button 'switcher'
+    fill_in('breadcrumbs-search', :with => 'three')
+    wait_for_ajax
```

Comment:
This test depends on this [PR](https://github.com/theforeman/foreman/pull/5541) which fixes the `wait_for_ajax ` **bug** (it didn't work with react).  Could we cp this PR as well?

Reply:
@amirfefer please add that commit to this PR

Reply:
@mmoll - done :)